src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.PackageDependencyResolution.targets

```diff
@@ -296,6 +298,8 @@ Copyright (c) .NET Foundation. All rights reserved.
         <Private>false</Private>
         <NuGetIsFrameworkReference>true</NuGetIsFrameworkReference>
         <NuGetSourceType>Package</NuGetSourceType>
+        <NuGetPackageId>%(_FrameworkAssemblies.PackageName)</NuGetPackageId>
```

Comment:
Question for my understanding (no change necessary):  Why the difference in metadata names here?  `NuGetPackageXXX` vs. just `PackageXXX`?  Why is one prefixed with `NuGet` and the other isn't?